test/internet/test-dns.js

```diff
@@ -390,6 +390,15 @@ TEST(function test_lookup_ipv6_explicit_object(done) {
 
 
 TEST(function test_lookup_ipv6_hint(done) {
+  // FreeBSD does not support V4MAPPED flag.
+  if (process.platform === 'freebsd') {
+    console.log(
+      '1..0 # Skipped: test_lookup_ipv6_hint is disabled on FreeBSD.'
```

Comment:
@thefourtheye yeah, not too much to do about "double skipping" unless we're keen to start rewriting larger parts of our tests. Long term we should look at replacing the current runner with something build on top of `node-tap` or similar. There's an issue about this somewhere..

anyway, moving forward; I'm ok with either message.